src/geostats.jl

```diff
@@ -46,6 +46,7 @@ Image quilting simulation solver as described in Hoffimann et al. 2017.
   @param inactive      = nothing
   @param soft          = []
   @param tol           = .1
+  @global device        = CPU1()
   @global threads      = cpucores()
   @global gpu          = false
```

Comment:
We should probably remove the gpu global option now that we are using device?

Reply:
I think it is no longer necessary. I removed it.